plugins/sql_functions.cc

```diff
@@ -47,6 +61,13 @@ SqlQueryFunctionConfigDescription()
              "SQL query to run.  The values in the dataset, as "
              "well as the input values, will be available for the expression "
              "calculation");
+    addField("output", &SqlQueryFunctionConfig::output,
+             "Controls how the query output is converted into a row. "
+             "'FIRST_ROW' (default) will return only the first row produced "
+             "by the query.  'NAMED_COLUMNS' will construct a row from the "
+             "whole returned table, which must have a 'value' column "
+             "containing the value.  If there is a 'column' column, it will "
+             "be used as a column name, otherwise the row name will be used.");
```

Comment:
We should populate the third parameter with 'FIRST_ROW' to better identify the default.